superbench/benchmarks/micro_benchmarks/gemm_flops_performance.py

```diff
@@ -106,10 +111,6 @@ def _preprocess(self):
                 self._commands.append(command)
 
         # TODO - To support more architecutres, currently only support compute capability = 7.0 or 8.0
-        capability = nv_helper.get_device_compute_capability()
-        if capability == 7.0:
-            self.__kernel_map['FP16_TC'] = 'cutlass_tensorop_h884gemm_256x128_32x2_*'
-
         if capability not in [7.0, 8.0]:
```

Comment:
Do we need to move all these check before command generation?

Reply:
To better test the command generation code, I kept the unsupported architecture checking after the command generation code. So we can test it on any platform.

Reply:
I think we should do it in the UT, not in the code. Correct?